docs/source/basics.md

```diff
@@ -143,7 +143,7 @@ and will cause the test to fail.
 an array was done like `thing.nested.0`. This will be deprecated in the
 1.0 release.
 
-It is also possible to save data using function calls, explained below.
+It is also possible to save data using function calls, [explained below](https://tavern.readthedocs.io/en/latest/basics.html#saving-data-from-a-response).
```

Comment:
```suggestion
It is also possible to save data using function calls, [explained below](#saving-data-from-a-response).
```

This can be a link to the anchor rather than a hyperlink :+1: